R/orderly.R

```diff
@@ -49,9 +54,13 @@ orderly_new <- function(name, template = NULL, force = FALSE,
                         root = NULL, locate = TRUE) {
   root <- root_open(root, locate, require_orderly = TRUE, call = environment())
   dest <- file.path(root$path, "src", name)
+  existing_entrypoint_filename <- find_entrypoint_filename(
```

Comment:
I don't think this is quite correct, because you'll mishandle the case where we have `src/foo/foo.R` and `src/foo/orderly.R` so we return `NULL` and think we can continue and then overwrite the existing file.

Reply:
should be able to verify this with your two-orderly-files test?